tools/myst/enc/enc.c

```diff
@@ -133,13 +133,24 @@ static uint64_t _forward_exception_as_signal_to_kernel(
     }
     if (oe_exception_code == OE_EXCEPTION_X87_FLOAT_POINT)
     {
-        siginfo.si_code = SI_KERNEL;
+        siginfo.si_code = FPE_FLTINV;
```

Comment:
I would suggest still keep the following comment so that we can improve this later.

ATTN: Consider implementing accurate si-code for OE_EXCEPTION_X87_FLOAT_POINT

Reply:
Makes sense, added the comment back.